lang/src/test/kotlin/org/partiql/lang/eval/NullIfEvaluationTest.kt

```diff
@@ -38,7 +38,6 @@ class NullIfEvaluationTest : EvaluatorTestBase() {
         fun testCase(expr1: String, expr2: String, expected: String) = NullIfTestCase(expr1, expr2, expected)
 
         @JvmStatic
-        @Suppress("unused")
```

Comment:
`@Suppress("unused")` on methods that create argument lists can now be removed in JUnit5 tests -- it looks like there are no longer warnings for this, if there ever were.